101-function-app-create-dedicated/azuredeploy.json

```diff
@@ -0,0 +1,104 @@
+{
+    "$schema": "http://schemas.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the function app that you wish to create."
+            }
+        },
+        "hostingPlanSettings": {
+            "type": "Object",
+            "defaultValue": {
+                "computeMode": "Dedicated",
+                "siteMode": "Limited",
+                "sku": "Standard",
+                "workerSize": "0",
+                "hostingEnvironment": ""
+            },
+            "metadata": {
+                "description": "The settings of the existing hosting plan."
+            }
+        },
+        "storageAccountName": {
```

Comment:
It's better to not ask the user for the storage account name :) See #944 for auto-generating this